app/public/js/system/guiConfig.js

```diff
@@ -28,7 +29,19 @@ guiConfig.maximize = function () {
 };
 
 guiConfig.logOut = function () {
-  fs.removeSync(`${__dirname}/userConfig.json`);
+  let _userConfigPath = `${__dirname}/app/public/js/system/userConfig.json`; // Windows specific for now
```

Comment:
in that case we should add a check if it's windows here too otherwise this will always be the case.